charts/zac/templates/config.yaml

```diff
@@ -43,7 +43,7 @@ data:
   {{- if .Values.opa.url }}
   OPA_API_CLIENT_MP_REST_URL: {{ .Values.opa.url }}
   {{- else }}
-  OPA_API_CLIENT_MP_REST_URL: http://{{ include "zaakafhandelcomponent.opa.fullname" . }}
+  OPA_API_CLIENT_MP_REST_URL: http://{{ include "zaakafhandelcomponent.opa.fullname" . }}:{{ .Values.opa.service.port }}
```

Comment:
Should we check that the port is set? or required in this situation to prevent it being empty or use a default?